client/src/components/EditProfile.js

```diff
@@ -183,11 +185,22 @@ class EditProfile extends Component {
 		      content='Make sure to fill in all starred fields'
 		    />;
 			this.setState({
-				error_message: cur_error
+				cur_error_message: cur_error
 			});
-		} else {
 
-            let skills = [this.state.skills_1, this.state.skills_2, this.state.skills_3]
+		} else if(this.checkProfanity()){
+            cur_error =
+            <Message>
+              <Message.Header>Your profile contains profanity</Message.Header>
```

Comment:
Ideally we could tell the user exactly where and what the problem is - at the very least it would be nice to indicate the word(s) that matched